pixeltable/datatransfer/pandas.py

```diff
@@ -0,0 +1,147 @@
+from typing import Optional, Any, Iterable
```

Comment:
Move this into utils/pandas.py and get rid of pixeltable/datatransfer, this feels like a utility.